### PR TITLE
feat: init

### PR DESCRIPTION
**Pull request by load1n9**

### CODE REVIEW

**Review by lino-levan — APPROVED** [review]
LGTM

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
   "deno.config": "./deno.json",
+  "deno.unstable": true,
```
Comment (MierenManz) [inline .vscode/settings.json:3]: We are currently not using anything unstable right?

File: core/client.ts [hunk 2]
```diff
@@ -10,19 +10,15 @@ import type { Server } from "@core/server.ts";
...
   id: number;
 
   constructor(conn: Deno.Conn, logger: Logger) {
     this.#inner = conn;
     this.#logger = logger;
-    this.state = State.HandShaking;
     this.id = conn.rid;
   }
 
-  /**
```
Comment (MierenManz) [inline core/client.ts:23]: why did this get removed?

**Review by MierenManz — CHANGES_REQUESTED** [review]
Mostly fine just one nit

File: core/client.ts (hunk 2 above)
Comment (lino-levan) [reply core/client.ts:23]: DMed loading about this after my LGTM.

File: .vscode/settings.json (hunk 1 above)
Comment (load1n9) [reply .vscode/settings.json:3]: ^

**Pull request merged**